tests/api2/test_130_cloudsync.py

```diff
@@ -1,11 +1,9 @@
-#!/usr/bin/env python3
-
 import pytest
 import sys
 import os
 import time
 import urllib.parse
-from pytest_dependency import depends
+
```

Comment:
We should opportunistically clean this test file up and convert to websocket.

Reply:
RIght now?

Reply:
Yeah, I don't see why we shouldn't. We're doing this in other PRs as well.